run_example,sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+if [ "$#" -ne 1 ]
+then
+    echo "Usage: $0 [android|ios]"
+    exit
+fi
+
+# Updates, builds, and loads the example app in an IDE.
+# All commands have been copied from the project's CircleCI workflow.
+
+# Update and build the root-level components..
+npm install
+npm build
+npm install -g @ionic/cli
+
+pushd example
+
+# Update and build the example app.
+npm install
+ionic build
+
+# Open example app in a Xcode or Android Studio.
```

Comment:
On Android this will run the app on an Android device or emulator, not Android Studio.

Reply:
Should I change the documentation, or the command?

Reply:
It needs both. I see the documentation change now. But would be good to move this comment down a line as well, since I believe the npx sync is part of the build as well.

Also, can you add a specific input check here, and if it is android, do `cd example ; ./gradlew assemble ; cd ..`?

Reply:
Done. Give it another try.